src/apps/subjects/crud/subject.py

```diff
@@ -58,6 +65,16 @@ async def get_by_user_ids(self, applet_id: uuid.UUID, user_ids: list[uuid.UUID])
         res = await self._execute(query)
         return res.scalars().all()
 
+    async def get_all_by_user_ids(self, applet_id: uuid.UUID, user_ids: list[uuid.UUID]) -> list[SubjectSchema]:
```

Comment:
Similar comment as above to replace the existing `get_by_user_ids` function

Reply:
Updated